pom.xml

```diff
@@ -45,17 +45,17 @@
 
     <properties>
         <json.schema.validator.version>2.2.14</json.schema.validator.version>
-        <jackson.version>2.10.1</jackson.version>
-        <jackson.databind.version>2.10.5.1</jackson.databind.version>
+        <jackson.version>2.13.1</jackson.version>
```

Comment:
we could import the jackson bom too (not sure it is worth it given we have just 2 jackson dependencies)